src/interface/interface.cpp

```diff
@@ -146,6 +146,7 @@ void initialize(std::string input_file_arg, int& problem_id, int& pts_per_cycle,
   interface->external_step_size_ = reader.sim_external_step_size;
 
   // For how many time steps are outputs being returned?
+  // NOTE: Only tested num_output_steps = interface->num_time_steps_
```

Comment:
Is this a limitation? Can you throw an error if this is not true?

Reply:
Done.